src/views/Resources.vue

```diff
@@ -1,57 +1,37 @@
 <template>
-  <base-card class="container">
-    <h1>Resources table</h1>
-    <resource-table :resources="resources"></resource-table>
-  </base-card>
+  <v-container class="my-12" fluid>
+    <v-row justify="center">
+      <v-col cols="10" max-width="1600">
+        <v-card class="elevation-12">
+          <div class="text-center">
+            <h1>Resources table</h1>
+          </div>
+          <resource-table></resource-table>
+        </v-card>
+      </v-col>
+    </v-row>
+  </v-container>
 </template>
 
 <script>
+import { onMounted } from "vue";
 import ResourceTable from "@/components/Tables/ResourceTable.vue";
-import { fetchResources } from "@/services/HttpClientService.js";
+import { useStore } from "vuex";
 
 export default {
   components: {
     ResourceTable,
   },
-  data() {
-    return {
-      resources: [],
-    };
-  },
+  setup() {
+    const store = useStore();
 
-  methods: {
-    async fetchResources() {
-      const fetchedResources = await fetchResources();
-      this.resources = fetchedResources;
-    },
-  },
-  mounted() {
-    this.fetchResources();
+    onMounted(() => {
+      store.dispatch("resources/fetchResources");
+    });
```

Comment:
We don't handle errors thrown by fetchResources here. I think we should like in the products.

Reply:
fixed